### ISSUE TITLE
question about training

### ISSUE THREAD

**Issue by toplinuxsir** [AlexeyAB/darknet#2792]
If I have trained 30000 Interations for 1000 images , But now I have another 200 images to add ,  Can I add the 200 images ( marked with tools yolo-mark )and restart the training from the last point ?
Thanks !

**Comment by drapado** [comment]
Yes, you can. Just add the files to the train.txt file and continue training with the last saved weigths in the backup folder:
`./darknet detector train ~data ~cfg ~_last.weights`

**Comment by toplinuxsir** [comment]
Thanks a lot

**Comment by toplinuxsir** [comment]
@drapado  If I add a class , For example : last trained data contains 5 classess, But I want to add one class and marked pics, Can I resume the training from last saved point ?
Thanks !
